tests/unit/reports/multi_table/test_multi_table_quality_report.py

```diff
@@ -85,6 +179,46 @@ def test_generate(self, mock_cardinality_score, mock_column_shapes_score,
         assert report._overall_quality_score == .5
         assert report._is_generated is True
 
+        mock_tqdm.assert_has_calls([
+            call(total=4, file=mock_sys),
+            call().set_description('(1/3) Evaluating Column Shapes: '),
+            call().close(),
+            call(total=2, file=mock_sys),
+            call().set_description('(2/3) Evaluating Column Pair Trends: '),
+            call().close(),
+            call(total=1, file=mock_sys),
+            call().set_description('(3/3) Evaluating Cardinality: '),
+            call().close()
+        ])
+
+    @patch(
+        'sdmetrics.reports.multi_table._properties.column_pair_trends.ColumnPairTrends.get_score')
+    @patch('sdmetrics.reports.multi_table._properties.column_shapes.ColumnShapes.get_score')
+    @patch('sdmetrics.reports.multi_table._properties.cardinality.Cardinality.get_score')
+    def test_generate_failed_scores(self, mock_cardinality_score, mock_column_shapes_score,
+                                    mock_column_pair_trends_score):
+        """Test the ``generate`` method when `get_score` for each property fails."""
+        # Setup
+        real_data, synth_data, metadata = self.get_data()
+        report = QualityReport()
+        mock_cardinality_score.side_effect = ValueError
+        mock_column_shapes_score.side_effect = ValueError
+        mock_column_pair_trends_score.side_effect = ValueError
+
+        # Run
+        report.generate(real_data, synth_data, metadata)
```

Comment:
can we either 
a. add a test where verbose is true that checks that `_print_results` gets called
or
b. add a test where verbose is true and just move the asserts from test__print_results into that test